contracts/DutchExchange.sol

```diff
@@ -19,6 +19,14 @@ contract DutchExchange {
         uint den;
     }
 
+    struct masterCopyCountdownType {
+        DutchExchange masterCopy;
+        uint timeWhenAvailable;
+    }
+
+    DutchExchange masterCopy;
```

Comment:
Are you sure that we can define it as DutchExchange? Because in proxy contract, it is only an address.
I don't know it